todo/src/components/Todos.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react'
+
+import { connect } from 'react-redux';
+
+import { addTodos, toggleTodos, deleteTodos } from '../action';
+
+class Todos extends Component {
+
+    state = {
```

Comment:
You will not need to store the todos on state, since they will be managed by redux